sql/nycha_communitycenters.sql

```diff
@@ -0,0 +1,40 @@
+--select w.status::text, count(*) 
+--from (select geo::json->'status' as status from bpl_libraries) w
+--group by w.status::text;
+
+ALTER TABLE nycha_communitycenters
+	ADD hash text, 
+	ADD	facname text,
+	ADD	factype text,
+	ADD	facsubgrp text,
+	ADD	facgroup text,
+	ADD	facdomain text, 
+	ADD	servarea text,
+	ADD	opname text,
+	ADD	opabbrev text,
+	ADD	optype text,
+	ADD	overagency text,
+	ADD	overabbrev text,
+	ADD	overlevel text,
+	ADD	capacity text,
+	ADD	captype text,
+	ADD	proptype text;
+
+update nycha_communitycenters as t
+SET hash =  md5(CAST((t.*)AS text)), 
+	facname = development,
+	factype = program_type,
+	facsubgrp = NULL,
```

Comment:
Based on the template, it should be `facsubgrp = 'Community Centers and Community School Programs',`